tools/distpartitioning/parmetis_preprocess.py

```diff
@@ -76,50 +97,50 @@ def gen_edge_files(rank, schema_map, params):
 
     # Regenerate edge files here.
     edge_data = schema_map[constants.STR_EDGES]
-    etype_names = schema_map[constants.STR_EDGE_TYPE]
-    etype_name_idmap = {e: idx for idx, e in enumerate(etype_names)}
 
     outdir = Path(params.output_dir)
     os.makedirs(outdir, exist_ok=True)
+
+    def process_and_write_back(data_df, idx):
+        data_f0 = data_df[:, 0]
+        data_f1 = data_df[:, 1]
+
+        global_src_id = data_f0 + ntype_gnid_offset[src_ntype_name][0, 0]
+        global_dst_id = data_f1 + ntype_gnid_offset[dst_ntype_name][0, 0]
+        cols = [global_src_id, global_dst_id]
+        col_names = ["global_src_id", "global_dst_id"]
+
+        out_file_name = Path(edge_data_files[idx]).stem.split(".")[0]
+        out_file = os.path.join(
+            outdir, etype_name, f"edges_{out_file_name}.csv"
+        )
```

Comment:
@kylasa note that we change the directory where edge files are stored, including the edge type in the directory structure. Previously if two edge types shared the same file names they'd end up overwriting each other's files.

So where previously for 
```
{
	"edge_type_1": 
		{"data": ["file_1.parquet", "file_2.parquet"]},
	"edge_type_2": 
		{"data": ["file_1.parquet", "file_2.parquet"]},
}
```
we'd end up with just two files under `f"{outdir}/edges_file_1.parquet"` and `f"{outdir}/edges_file_2.parquet"`,
now we get the correct: `f"{outdir}/edge_type_1/edges_file_1.parquet"`, `f"{outdir}/edge_type_1/edges_file_2.parquet"` etc.